pyartcd/pyartcd/pipelines/report_rhcos.py

```diff
@@ -0,0 +1,138 @@
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+import click
+import asyncio
+from jenkinsapi.jenkins import Jenkins
+from jenkinsapi.custom_exceptions import NoBuildData
+from subprocess import run
+
+
+class CheckRhcosPipeline:
+    def __init__(self, runtime: Runtime, channel: str):
+        self.runtime = runtime
+        self.channel = channel
+        self.result = dict()
+
+        self.versions = [
+            "4.6",
+            "4.7",
+            "4.8",
+            "4.9",
+            "4.10",
+            "4.11"
+        ]
+
+        self.arches = [
+            "amd64",
+            "s390x",
+            "ppc64le",
+            "arm64"
+        ]
+
+        self.rhcos_urls = {
+            'amd64': 'https://jenkins-rhcos-art.cloud.privileged.psi.redhat.com',
+            's390x': 'https://jenkins-rhcos.cloud.s390x.psi.redhat.com',
+            'ppc64le': 'https://jenkins-rhcos.cloud.p8.psi.redhat.com',
+            'arm64': 'https://jenkins-rhcos.cloud.arm.psi.redhat.com'
+        }
+
+    async def get_data(self):
+        for arch in self.arches:
+            self.result[arch] = dict()
+            self.result[arch] = await self.get_data_for_arch(arch)
+
+    async def get_data_for_arch(self, arch):
+        jenkins = Jenkins(self.rhcos_urls[arch])
+
+        a = dict()
+        for version in self.versions:
+            if arch == 'arm64' and int(version.split('.')[-1]) < 9:
```

Comment:
A comment would be appreciated :) even if I can tell "arm is not applicable for OCP < 4.9)